editor/header/preview-button/index.js

```diff
@@ -73,6 +73,16 @@ export class PreviewButton extends Component {
 			'about:blank',
 			this.getWindowTarget()
 		);
+
+		const popupLoader = renderToString(
+			<div>
+				<p>Please wait&hellip;</p>
+				<p>Generating preview.</p>
+			</div>
+		);
+		const css = '<style type="text/css"> div { margin-top: 25%; } p { text-align: center; } </style>';
```

Comment:
`type="text/css"` is not necessary since it is the default:

>This attribute is optional and defaults to `text/css` if it's missing.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style